egs/multi_en/s5/local/make_partitions.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright 2016  Allen Guo
+# Apache License 2.0
+
+# This script creates the data directories that will be used during training.
+# This is discussed fully in README.md, but the gist of it is that the data for each
+# stage will be located at data/{MULTI}/{STAGE}, and every training stage is individually
+# configurable for maximum flexibility.
+
+# Note: The $stage if-blocks use -eq in this script, so running with --stage 4 will
+# run only the stage 4 prep.
+
+multi=multi_a  # This defines the "variant" we're using; see README.md
+stage=1
+
+. utils/parse_options.sh
+
+data_dir=data/$multi
+tmp1=$data_dir/tmp1
```

Comment:
As `tmp1` and `tmp2` are not deleted at the end of the script, I would suggest the use of more understandable names.